includes/class-wc-stripe-helper.php

```diff
@@ -516,183 +476,33 @@ public static function get_legacy_payment_method( $id ) {
 	 * The ids are mapped to the corresponding equivalent UPE method ids for rendeing on the frontend.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
 	 */
 	public static function get_legacy_available_payment_method_ids() {
-		$stripe_settings            = self::get_stripe_settings();
-		$payment_method_classes     = self::get_legacy_payment_method_classes();
-		$ordered_payment_method_ids = isset( $stripe_settings['stripe_legacy_method_order'] ) ? $stripe_settings['stripe_legacy_method_order'] : [];
-
-		// If the legacy method order is not set, return the default order.
-		if ( ! empty( $ordered_payment_method_ids ) ) {
-			$payment_method_ids = array_map(
-				function ( $payment_method_id ) {
-					if ( 'stripe' === $payment_method_id ) {
-						return WC_Stripe_Payment_Methods::CARD;
-					} else {
-						return str_replace( 'stripe_', '', $payment_method_id );
-					}
-				},
-				$ordered_payment_method_ids
-			);
-
-			// Cover the edge case when new Stripe payment methods are added to the plugin which do not exist in
-			// the `stripe_legacy_method_order` option.
-			if ( count( $payment_method_ids ) - 1 !== count( $payment_method_classes ) ) {
-				foreach ( $payment_method_classes as $payment_method_class ) {
-					$id = str_replace( 'stripe_', '', $payment_method_class::ID );
-					if ( ! in_array( $id, $payment_method_ids, true ) ) {
-						$payment_method_ids[] = $id;
-					}
-				}
-
-				// Update the `stripe_legacy_method_order` option with the new order including missing payment methods from the option.
-				$stripe_settings['stripe_legacy_method_order'] = $payment_method_ids;
-				self::update_main_stripe_settings( $stripe_settings );
-			}
-		} else {
-			$payment_method_ids = array_map(
-				function ( $payment_method_class ) {
-					return str_replace( 'stripe_', '', $payment_method_class::ID );
-				},
-				$payment_method_classes
-			);
-			$payment_method_ids = array_merge( [ WC_Stripe_Payment_Methods::CARD ], $payment_method_ids );
-		}
-
-		return $payment_method_ids;
+		return [];
 	}
 
 	/**
 	 * List of enabled legacy payment methods.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
```

Comment:
Nit: Another version fix.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in future versions.
```